docs/.vuepress/config.js

```diff
@@ -41,6 +41,41 @@ module.exports = {
         },
       },
     ],
+    ['@simonbrunel/vuepress-plugin-versions', {
+      filters: 'version-filters.js',
+      menu: {
+        items: [
+          {
+            text: 'Documentation',
+            items: [
+              {
```

Comment:
Is there a way to detect when an RC exists > than the current version and insert it between `master` and `latest`? Or will the RC appear as the latest?

Reply:
Should work automatically, right @simonbrunel ?

Reply:
The `alpha`/`beta`/`rc` should appear between `master` and `latest` with, if set in npm, the `next` suffix.

Reply:
Example with [TypeScript](https://www.npmjs.com/package/typescript)

![image](https://user-images.githubusercontent.com/3874900/114272388-4440fd80-9a16-11eb-90e9-de6f5f7cdbb8.png)

And [VuePress](https://www.npmjs.com/package/vuepress)

![image](https://user-images.githubusercontent.com/3874900/114272408-591d9100-9a16-11eb-9b80-c5151f294d63.png)